.circleci/config.yml

```diff
@@ -718,6 +729,11 @@ workflows:
           filters:
             tags:
               only: /.*/
+      # Check CHANGELOG update
+      - check-changelog:
```

Comment:
on arnold you ignore the master branch. Why not here ?

Reply:
Because I forgot. Thank you :heart: